### PR TITLE
[FSDE-11701] Migrate Link component to new Vue documentation

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**

### CODE REVIEW

File: src/documentation/pages/components/link/examples/_base.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" :tabs="exampleTabs">
+    <Wrapper slot="example">
+        <chi-link href="#">Link</chi-link>
```
Comment (evansito) [inline src/documentation/pages/components/link/examples/_base.vue:4]:
```suggestion
      <chi-link href="#">Link</chi-link>
```

**Review by evansito — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://04e181be0eedd995b9fece6e8f28443944242ec7-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-268-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://2394aaec4e8cdd5bc17a628d2016670325ebc526-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-268-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/documentation/pages/components/link/examples/_additional-sizes.vue [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<template lang="pug">
+  <ComponentExample title="Additional Sizes" id="additional-sizes" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_additional-sizes.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_base.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_base.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_cta.vue [hunk 4]
```diff
@@ -0,0 +1,49 @@
+<template lang="pug">
+  <ComponentExample title="CTA" id="cta" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_cta.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_disabled.vue [hunk 5]
```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+  <ComponentExample title="Disabled" id="disabled" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_disabled.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_download.vue [hunk 6]
```diff
@@ -0,0 +1,57 @@
+<template lang="pug">
+  <ComponentExample title="Download" id="download" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_download.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_hreflang.vue [hunk 7]
```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+  <ComponentExample title="Hreflang" id="hreflang" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_hreflang.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_icon-link.vue [hunk 8]
```diff
@@ -0,0 +1,56 @@
+<template lang="pug">
+  <ComponentExample title="Icon Link" id="icon-link" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_icon-link.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_labeled-icon-link.vue [hunk 9]
```diff
@@ -0,0 +1,58 @@
+<template lang="pug">
+  <ComponentExample title="Labeled Icon Link" id="labeled-icon-link" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_labeled-icon-link.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_no-hover-underline.vue [hunk 10]
```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+  <ComponentExample title="No Underline on Hover" id="noHoverUnderline" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_no-hover-underline.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_rel.vue [hunk 11]
```diff
@@ -0,0 +1,48 @@
+<template lang="pug">
+  <ComponentExample title="Rel" id="rel" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_rel.vue:2]: please add additional class `-text`

File: src/documentation/pages/components/link/examples/_target.vue [hunk 12]
```diff
@@ -0,0 +1,75 @@
+<template lang="pug">
+  <ComponentExample title="Target" id="target" :tabs="exampleTabs">
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/_target.vue:2]: please add additional class `-text`

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ef173747088b7bc4eadb0048d47faa1dcd3d64ed-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-268-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/4/. :x:

File: src/documentation/pages/components/link/examples/_no-hover-underline.vue [hunk 13]
```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+  <ComponentExample title="No Underline on Hover" id="noHoverUnderline" :tabs="exampleTabs" additionalClasses="-text">
```
Comment (evansito) [inline src/documentation/pages/components/link/examples/_no-hover-underline.vue:2]: i'd recommend using the html convention of using dashes - for ids like in your other examples, e.g. no-hover-underline

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/6/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://27802a0ef7fe725db5ed0ecc5f684d9784ea1a40-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-268-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://99dd6f6d8c33b969c703a7210b7b3e68df384f44-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-268-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/13/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://16e0d8e3a1ca09b0fedd87d6192dba2f0a0f1a06-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-268-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/15/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/16/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/17/. :x:

File: src/documentation/pages/components/link/_accessibility.vue [hunk 14]
```diff
@@ -0,0 +1,16 @@
+<template lang="pug">
+  <div>
```
Comment (jllr) [inline src/documentation/pages/components/link/_accessibility.vue:2]:
```suggestion
  div
```

File: src/documentation/pages/components/link/_accessibility.vue [hunk 15]
```diff
@@ -0,0 +1,16 @@
...
+    h2 Accessibility
+    .chi-card.-align--center.-empty
+      .chi-card__content
+        .chi-card__caption
+          | Accessibility guidelines coming soon
+  </div>
```
Comment (jllr) [inline src/documentation/pages/components/link/_accessibility.vue:8]:
```suggestion
```

File: src/documentation/pages/components/link/_properties.vue [hunk 16]
```diff
@@ -0,0 +1,13 @@
+<template lang="pug">
+  <div>
```
Comment (jllr) [inline src/documentation/pages/components/link/_properties.vue:2]:
```suggestion
  div
```

File: src/documentation/pages/components/link/_properties.vue [hunk 17]
```diff
@@ -0,0 +1,13 @@
...
+    h2 Web Component
+    <PropertiesGenerator tag="chi-link" />
+  </div>
```
Comment (jllr) [inline src/documentation/pages/components/link/_properties.vue:5]:
```suggestion
```

File: src/documentation/pages/components/link/examples/index.vue [hunk 18]
```diff
@@ -0,0 +1,49 @@
+<template lang="pug">
+  <div>
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/index.vue:2]:
```suggestion
  div
```

File: src/documentation/pages/components/link/examples/index.vue [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+    <NoHoverUnderline />
+    <CTA />
+    <LabeledIconLink />
+    <IconLink />
+    <Target />
+    <Rel />
+    <Download />
+    <Hreflang />
+    <AdditionalSizes />
+  </div>
```
Comment (jllr) [inline src/documentation/pages/components/link/examples/index.vue:16]:
```suggestion
```

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/18/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://27ea56d75d59f7140f8d23042f6114430c9ea1a1-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-268-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-268/20/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://e8d93855c24ec1beb6aba1fbcda3fbfa852be203-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-268-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Pull request merged**